app/views/root/local_transaction.html.erb

```diff
@@ -58,3 +58,17 @@
     </div>
   </section>
 <% end %>
+
+<% if @postcode_error %>
+  <script type="text/javascript">
+    //<![CDATA[
+      GOVUK.analytics.trackEvent(
+        "userAlerts:LocalTransaction",
+        "postcodeErrorShown:<%= @postcode_error %>",
```

Comment:
The format of the category and action being passed to GA is different to many of the existing events. It suggests a type of namespacing that doesn't currently exist, is this the intent? Will there be similar events added later?

The preference for category is a human readable string, eg "External Link Tracked", or "Print Intent". Perhaps something like "Postcode Error".

I'm not sure the `LocalTransaction` bit is needed, this is identified using a Custom Dimension via:
`<meta name="govuk:format" content="local_transaction">`